routers/user/auth.go

```diff
@@ -1433,3 +1434,17 @@ func MustChangePasswordPost(ctx *context.Context, cpt *captcha.Captcha, form aut
 
 	ctx.Redirect(setting.AppSubURL + "/")
 }
+
+// BuildPasswordComplexityError builds the error message when password complexity checks fail
+func BuildPasswordComplexityError(ctx *context.Context) string {
```

Comment:
Does it make more sense to put this func closer to where the password complexities are set rather than in a router?

Reply:
@jolheiser I was afraid I would get a circular reference because of `modules/context`, but it turns out I don't. Fixed.